### ISSUE TITLE
Content: attitudinal changes

### ISSUE THREAD

**Issue by chendaniely** [swcarpentry/assessment#3]
What do learners _feel_

Example:

Learners should feel that using the shell can increase their productivity

**Comment by chendaniely** [comment]
likert scale questions:

shell: I can see how the shell can simplify my work
shell: I can see how using the shell can make me more productive
shell: The shell will allow me to navigate local filesystems quicker
shell: The shell will allow me to navigate remote filesystems quicker
shell: The shell can help me automate my pipelines
shell: My programs should be broken into small functions that do one thing, and one thing well

**Comment by xuf12** [comment]
@chendaniely how do we define attitudinal changes? do we use the Likert value difference or something from Rasch? Would Likert scale mean different things when the background info of participants are diverse?
what's your opinion on creating pre-post paired questions without using the Likert scale (use yes or no instead)? The example pre-post questions below may be linked with a few background questions on what kind of work they do (% of time spent on remote filesystems, whether dealing with lots of files, etc.)

Shell:
(pre) I don't know how to use the shell to manage my files or automate my analytic pipelines, and I'd like to learn it. 
(post) I feel confident to use the shell to manage my files and automate my analytic pipelines.

(pre) I am already productive at work without using the shell.
(post) I am even more productive at work with using the shell.

(pre) I think learning the shell will make my work more productive.
(post) I will continue using the shell as it clearly makes my work more productive.

(pre) I don't know the best way to learn the shell but I'd like to learn it.
(post) I feel comfortable with the basics of the shell and will learn more on my own.

**Comment by chendaniely** [comment]
@xuf12 we could do pre/post questions but given the number of non-respondents it's unlikely we can get pre-post measurements from the same person.  A lot of data may be lost if we try to tie those questions together.

We _could_ ask the pre-post in 1 question in the post survey like: I feel confident to use the shell to manage my files and automate my analytic piplines, more so than before attending the workshop (or something similar)

**Comment by tracykteal** [comment]
Shari Ellis at iDigBio advised us to do those types of questions, so we're assessing after versus before the workshop in one question, as:

"Compared to before the workshop, I have a better understanding of how to:"

![screen shot 2015-06-03 at 2 37 11 pm](https://cloud.githubusercontent.com/assets/889238/7968327/1291c8d6-09fe-11e5-8b11-e27c8d2d353d.png)

**Comment by xuf12** [comment]
@tracykteal @chendaniely Thanks for the examples. I guess we can use the objectives in each subject of the SWC lessons to ask these questions?

**Comment by xuf12** [comment]
So for all the questions below, I'd use the 5 options from @tracykteal as answers
All of them start with this sentence:
"Compared to before the workshop..."

Shell:
I have a better understanding of how to use the Shell to manage my files and directories.
I have a better understanding of how to use the Shell to automate repeated tasks.
I have a better understanding of how to use the Shell to find things.

Whenever I need to access files on my computer, I open a terminal as opposed to using a GUI-based file explorer.
I think the Shell has made my work more productive, and I would like to learn more about it on my own.

Git:
I have a better understanding of how to use Git to version control files on my computer.
I have a better understanding of how to use an online Git repository hosting service (e.g. Github) to version control files in collaborative projects.

I think of version control whenever I write code that I want to use for the future.

**Comment by drlabratory** [comment]
One problem with the "better understanding" language is that students in general don't understand their own levels of understanding. The questions are good, but I would replace "have a better understanding of how to" with "can"

**Comment by xuf12** [comment]
@drlabratory I completely agree with you on that. However, I think a self-reported answer here is a sufficiently accurate reflection of their attitude, and that's what we are measuring. Answers here should be analyzed together with those from the skills and declarative knowledge sections, which ideally echo each other in some ways.
